src/App.tsx

```
@@ -145,6 +146,7 @@ function App() {
           <>
             <h1 className="text-xl text-center">What's your mode today 🤩 ?</h1>
             <div className="flex gap-2 mx-auto w-fit">
+              {ankit.map(item=><h1>{item.value}</h1>)}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)